apps/examples/rssi_report/rreport_http.c

```diff
@@ -0,0 +1,73 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
```

Comment:
I think we don't need this.

Reply:
could you elaborate it? I can't understand.

Reply:
`<tinyara/config.h>` is needed only if there is `CONFIG_XXX` in this file. 
But I think there is no in this file.

Reply:
curl.h contains CONFIG_XXX so adding tinyara/config.h is safer.